.github/workflows/cleanup-sdlc.yml

```
@@ -0,0 +1,222 @@
+            # Remove lines starting with sdlc.
+            sed -i '/^sdlc\./d' quarkus-app/src/main/resources/application.properties
+          fi
+
+      - name: Update README notice
+        run: |
+          echo "Updating README migration notice..."
+          if grep -q "AI-SDLC Migration Notice" README.md; then
+            # Replace the notice with a simple "completed" message
+            sed -i '/## ⚠️ AI-SDLC Migration Notice/,/^---$/c\
+## ✅ AI-SDLC Migration Completed\
+\
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)